components/datatypes/consent-preferences.schema.json

```diff
@@ -113,46 +154,6 @@
           "description": "User's Direct Marketing Preferences",
           "type": "object",
           "properties": {
-            "xdm:preferred": {
```

Comment:
@RussStringham - Can you please deprecate these properties instead of deleting them .

Reply:
These properties aren't being (fully) deleted. The actual XDM is defined using ref macros, and they are being incorporated into the same area from a different macro. With the original definition, the macros exposed these fields in two places. However, they should only be used in one of those places. This update makes sure they are only exposed in that one place (they have never been populated with data in the other place). See my 2nd comment in the "conversations" section of this pull request for more details.